docs/conf.py

```diff
@@ -48,23 +57,36 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
+autodoc_mock_imports = ["numba"]
 
 # -- Options for HTML output -------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_rtd_theme"
+html_theme = "sphinx_book_theme"
+
+# Config for the Sphinx book
+
+html_baseurl = "https://boost-histogram.readthedocs.io/en/latest/"
+
+
+html_theme_options = {
+    "home_page_in_toc": True,
+    "repository_url": "https://github.com/scikit-hep/boost-histogram",
```

Comment:
```suggestion
    "repository_url": "https://github.com/scikit-hep/vector",
```

You can seem my starting point. :)